### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by lepiaf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
```
Comment (Spomky) [inline composer.json]: `"php": ">=7.2"`

**Review by Ciloe — APPROVED** [review]

**Comment by Ciloe** [comment]
Is it possible to advance on this PR ?

**Comment by toooni** [comment]
@lepiaf  Can you make something like `git commit --allow-empty -m "Force CI"`. It seems like the travis failed for unknown reasons.

**Comment by bocharsky-bw** [comment]
Seems because of the limit hit Travis CI does not work, but tests pass for me locally on PHP 8 and because changes are simple here - I'm going to merge it

Thank you, guys!

**Comment by Kocal** [comment]
Nice!
Thanks everyone! :)

**Pull request merged**